__tests__/app.test.js

```diff
@@ -245,3 +245,34 @@ describe('POST /api/articles/:article_id/comments', () => {
 			})
 	})
 })
+describe('DELETE /api/comments/:comment_id', () => {
+	test('DELETE:204 responds with 204 no content, removes the passed comment from the database', () => {
+		return request(app)
+			.delete('/api/comments/1')
+			.expect(204)
+			.then(() => {
+				return db.query(`SELECT * FROM comments`).then(({ rows }) => {
+					const verified = rows.some((comment) => {
```

Comment:
Nice use of db.query to check that the item is no longer there! No required, but personally I would have this in!! Great work 👍